assets/sass/components/global/_googlesitekit-header.scss

```
@@ -117,6 +117,18 @@
 				padding-right: $grid-gap-desktop;
 			}
 
+			.googlesitekit-header__date-range-selector-menu--has-unified-dashboard {
+
+				&.mdc-button--dropdown {
+
+					@media (max-width: $bp-mobileOnly) {
+
+						background: none;
+						padding-right: 8px;
```

@techanvil one more suggestion - let's use `$grid-gap-phone * 0.5` instead of hardcoding `8px` here because that's how we define the margin for the SVG element in the date range picker (see a few lines above).
```suggestion
						padding-right: $grid-gap-phone * 0.5;
```

---

Sounds sensible. I've pushed that change too, thanks.